README.md

```diff
@@ -100,9 +128,26 @@ secret/kcp-syncer-kind-25coemaz created
 deployment.apps/kcp-syncer-kind-25coemaz created
 ```
 
+Now you need to use `kubectl kcp bind compute` to let `kcp` know about this new `kind` cluster that is providing compute resources. You do that by running: 
```

Comment:
The `bind compute` command should be done from **inside** the new workspace used to deploy applications.

Reply:
@davidfestal I think because of the flow that's the case.. running the steps as described here was working as expected.